src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1460,6 +1460,22 @@
     /**
      * A right parenthesis ({@code )}).
      *
+     * <p>For example:</p>
+     * <pre> 
+     * check();
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * |--METHOD_DEF -&gt; METHOD_DEF
```

Comment:
it is not method definition (METHOD_DEF), it is method invocation (METHOD_CALL)
please correct your ast tree and update PR description

Reply:
Commit 8bec077 had made the requested changes.